### ISSUE TITLE
Create custom button

### ISSUE THREAD

**Issue by ashishnagpal2498** [Tejas1510/Athavani#125]
**Is your feature request related to a problem? Please describe.**
Create a custom "Browse" button for better UI look.
![Inkedathavani_issue1_2](https://user-images.githubusercontent.com/36301481/111066706-71b58c80-84e6-11eb-808e-9840186fb0c1.jpg)

**Comment by ShubhamSj07** [comment]
@ashishnagpal2498 , I would like to take this issue
Also, pls provide me dummy credentials for login
Thanks

**Comment by sanjay270899** [comment]
@ShubhamSj07 
Go Ahead!
Use this Link.
https://athavani-sanjay-frontend.netlify.app/

**Comment by ShubhamSj07** [comment]
@sanjay270899 , how to set-up server locally, I am not able to set-up back end .
To make changes , I need to sign in
Plz help me out!

**Comment by AmeyaJain-25** [comment]
Hello @sanjay270899  @ashishnagpal2498 ,
Actually this issue is been solved under issue #123 
I have made the PR #134 for it.
Issue was for the UI and VALIDATION of CreatePost. I had mentioned too that I would be resolving the UI for choose a file button.

While going through issues, I came to know that this issue is already done in my PR.
Here's the screenshot for the button to upload the photos.
![Screenshot from 2021-03-18 00-46-26](https://user-images.githubusercontent.com/66305085/111530801-c0cf1c00-8789-11eb-86ca-e0ddb47ccd80.png)
